src/index.html

```diff
@@ -17,16 +17,35 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="wrap">
+      <form
+        class="form big--form"
+        action="#"
+        method="get"
+        data-qa="big"
+      >
+        <div class="icon big--icon"></div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+        <input
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          class="text big__text"
+        />
+      </form>
+      <form
+        class="form small--form"
+        action="#"
+        method="get"
+        data-qa="small"
+      >
+        <div class="icon small--icon"></div>
```

Comment:
Class names should represent the meaning of the content. The classes `small--form` and `small--icon` seem to describe the size rather than the content or function.